js/cell_toucher.js

```diff
@@ -19,70 +19,31 @@
  along with Minesweeper.js.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-var self = this;
-
-function postDebug(obj) {
-    var data = {
-        'type': 'log',
-        'obj': obj
-    };
-    if (self && self.postMessage) {
-        self.postMessage(JSON.stringify(data));
-    }
-}
 
 /**
  * @param obj
  * @param grid
  * @returns {Array}
  */
-function get8AdjacentSquares(obj, grid) {
-    var array = [],
+function getAdjacentSquares(obj, grid) {
+	    var Gij,
```

Comment:
could you change Gij name to somthing more readable. Good job reducing the code in this function though.